jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -262,6 +262,8 @@ def _get_cube_value(self, image, arr, x, y, viewer):
             # cubeviz case:
             return arr[int(round(x)), int(round(y)), viewer.state.slices[-1]]
         elif image.ndim == 2:
+            if isinstance(viewer, RampvizImageView):
+                x, y = y, x
```

Comment:
Just for clarity, swapping here has to do with the ramp cube convention? Also, can the L264 and L265 conditions be combined or is there going to be an additional case in the future that also has a 2 dimensional image?

Reply:
Full honesty – I'm not sure why the ramp cube coordinates seem to need a transpose here for coordinate info. There are already places in the logic in `coords_info` where we index the mouseover pixel by `arr[x, y, ...]` or `arr[y, x, ...]`, and I'm not positive I understand what's going on there.

Reply:
I keep the `RampvizImageView` case separate from `ImvizImageView` viewers with `image.ndim == 2` because Imviz works fine as is.